### ISSUE TITLE
Question on QR-DQN calculate_quantile_huber_loss

### ISSUE THREAD

**Issue by kentropy** [ku2482/fqf-iqn-qrdqn.pytorch#16]
When calculating the quantile huber loss in QR-DQN ([here](https://github.com/ku2482/fqf-iqn-qrdqn.pytorch/blob/master/fqf_iqn_qrdqn/utils.py#L42)), the whole term `torch.abs(taus[..., None] - (td_errors.detach() < 0).float()) * element_wise_huber_loss` is divided by `self.kappa`.
I cannot find this equation in the paper. Is there any reason for this implementation?

**Comment by toshikwa** [comment]
I referred to [IQN paper](https://arxiv.org/abs/1806.06923). However, there is no difference because `kappa = 1.0`.
Thanks.
